src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -10228,6 +10229,7 @@ void InitializeOperatingMode(EnergyPlusData &state,
             //         now check to see if coil is scheduled off
             if (GetCurrentScheduleValue(state, state.dataHVACVarRefFlow->TerminalUnitList(TUListNum).CoolingCoilAvailSchPtr(NumTU)) > 0.0) {
                 state.dataHVACVarRefFlow->TerminalUnitList(TUListNum).CoolingCoilAvailable(NumTU) = true;
+                state.dataHVACVarRefFlow->VRFTU(TUListNum).coolingCoilActive = true;
```

Comment:
You need to use the TU index here:
state.dataHVACVarRefFlow->VRFTU(TUIndex).coolingCoilActive

Reply:
Done.